extensions/qute/runtime/src/main/java/io/quarkus/qute/runtime/BuiltinTemplateExtensions.java

```diff
@@ -27,13 +28,17 @@ public static Object map(Map map, String name) {
             case "empty":
             case "isEmpty":
                 return map.isEmpty();
-            case "get":
-                return map.get(name);
-            case "containsKey":
-                return map.containsKey(name);
             default:
-                return Result.NOT_FOUND;
+                return map.containsKey(name) ? map.get(name) : Result.NOT_FOUND;
```

Comment:
You can also use `map.getOrDefault(name, Result.NOT_FOUND);` here

Reply:
@mkouba are you going to apply this or not?

Reply:
Sure, why not. I'll update the PR in a minute.